pkg/config/env/gcp.go

```diff
@@ -6,12 +6,18 @@ import (
 	"github.com/gobuffalo/envy"
 )
 
-// GcpBucketName returns Google Cloud Storage bucket name defined by ATHENS_STORAGE_GCP_BUCKET
-func GcpBucketName() (string, error) {
+// GCPBucketName returns Google Cloud Storage bucket name defined by ATHENS_STORAGE_GCP_BUCKET
+func GCPBucketName() (string, error) {
 	env, err := envy.MustGet("ATHENS_STORAGE_GCP_BUCKET")
 	if err != nil {
 		return "", fmt.Errorf("missing Google Cloud storage bucket name: %s", err)
 	}
 
 	return env, nil
 }
+
+// GCPPRojectID returns the project id on which the project
+// is running or the cloud storage is using.
+func GCPPRojectID() string {
+	return envy.Get("GCP_PROJECT_ID", "")
```

Comment:
Should this be documented anywhere (OK if the answer is yes, but we don't know where, yet.)

Reply:
It should, Maybe in the above with https://github.com/gomods/athens/pull/380#discussion_r207253615

Reply:
you mean the GCP set up in general or this environment variable? If the first then, yes I was thinking of having a README in every storage implementation that shows you its set up and why we have that implementation in the first place.

Reply:
A bit of both. Sounds good to me.

Reply:
:+1:

Reply:
m: capital R